backend/src/org/commcare/util/SessionNavigationResponder.java

```diff
@@ -0,0 +1,22 @@
+package org.commcare.util;
```

Comment:
`util` is sort of a strange place to put this. Is there a session related folder that this, and related files, can be put in? If not, let's create one.